github/github_test.go

```diff
@@ -135,32 +134,26 @@ func testBody(t *testing.T, r *http.Request, want string) {
 	}
 }
 
-// Helper function to test that a value is marshalled to JSON as expected.
+// Test whether the marshaling of v produces a JSON that corresponds
```

Comment:
Let's please say "produces JSON" instead of "produces a JSON".

Reply:
done!